pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -77,7 +77,25 @@ This procedure describes how to create a key/value pair secret called `my-secret
    dcos security secrets create --value=top-secret developer/my-secret
    ```
 
-# Creating secrets from a file via the DC/OS Enterprise CLI
+# Creating secrets from a binary file via the DC/OS Enterprise CLI
+
+This procedure describes how to use any file to create a secret called `myapp.keystore` inside the `developer` path using the DC/OS Enterprise CLI.
+
+The contents of the file (referred to below as `myapp.keystore`) can be any value. The secrets service encodes binary files into `BASE64` text form before storing it in secret store.
+
+**Note:** As of DC/OS 1.10, you can only upload a secret as a file from the DC/OS CLI. The maximum file size for a secret is approximately one MiB, subtracting approximately one KB for the secret store metadata.
```

Comment:
> As of DC/OS 1.10, you can only upload a secret as a file from the DC/OS CLI.

This means that there was an API / GUI option before?
Or is it that a secret can only be a file, and before it could be something else?